src/ResourceManager/Resources/Commands.Resources/ChangeLog.md

```diff
@@ -26,6 +26,8 @@
 
 ## Version 6.4.1
 * Fixed issue with default resource groups not being set.
+* Update New-AzureRmResourceGroupDeployment with new parameter RollbackAction
```

Comment:
Please move this to the "Current Release" header to ensure this will be released.